src/config.template.cjs

```diff
@@ -289,7 +289,7 @@ module.exports = {
 	 * Combine this with "excludeRecentSearch" and "searchCadence" to smooth long-term API usage patterns.
 	 * Default is no limit.
 	 */
-	searchLimit: undefined,
+	searchLimit: 100,
```

Comment:
For onboarding, I'm not sure that this would really please everyone. The general response I've gotten is people want the fastest possible search, and doing 100 a day initially (which would be the most common place these settings would be used prior to customizations) would make this a rather long process for many users initially.

Reply:
I think this is an opportunity for us to be opinionated and friendly to the trackers. It won't satisfy everyone, but if impatient users want it to do everything at once, they can still change the settings. Many users _will_ be okay with it, and it'll lead to better experiences in the long run.